src/timezones/types.jl

```diff
@@ -3,6 +3,21 @@
 using Base.Dates
 import Base.Dates: value
 
+
+abstract TimeError <: Exception
+
+type AmbiguousTimeError <: TimeError
+    dt::DateTime
+    tz::TimeZone
+end
+Base.showerror(io::IO, e::AmbiguousTimeError) = print(io, "Local DateTime $(e.dt) is ambiguious");
```

Comment:
Is `e.dt` going to show the local datetime because this is thrown when trying to construct a `ZonedDateTime`? (just making sure there aren't other places where we'd through this where `dt` is UTC)